changelog

```diff
@@ -1,3 +1,456 @@
+      * Significantly increase the energy capacity of Archons. (@Zitchas)
+      * Nerfed the flamethrower. (@Quantumshark)
+        * +100% firing heat (2 -> 4).
+        * -25% heat damage (200 -> 150)
+        * Added damage dropoff with range beyond 45 units. Damage is halved by maximum range.
+      * Increased the size of some Large Syndicate fleet variants. (@Quantumshark)
+      * Added a small amount of inertia reduction and force protection to Quarg engines. (@Zitchas)
+      * Made the Unfettered Hai attacking Wah Ki cowards and the Hai defending the system merciful, so fewer Unfettered get disabled or die there. (@Hurleveur)
+      * Slightly buffed the Berserker and Hawk. (@Zitchas)
+        * Berserker:
+          * Mass: 110 -> 90
+        * Hawk:
+          * Mass: 150 -> 110
+          * Required crew: 2 -> 1
+      * Reduced the lifetime of the Hai Tracker from 600 to 475 (range reduced from 8400 to 6650). (@Zitchas)
+      * Buffed the Hai Wiliwaw Cooling. 570 -> 660 cooling. (@Zitchas)
+      * Buffed the Hai Grasshopper. (@Hurleveur)
+        * Outfit space: 169 -> 198
+        * Weapon capacity: 38 -> 47
+        * Some adjustments to outfitting and a new variant carrying an Ion Cannon.
+      * Adjusted the Quicksilver. (@Zitchas)
+        * Reduce chassic cost: 1090000 -> 836000
+        * Increased cargo space: 10 -> 15
+        * Removed the Cooling Ducts and use a cheaper generator.
+      * Reduce the 0.9.15 afterburner buff. (@Amazinite)
+        * Reduced thrust by 20-30% and increased fuel usage by 20-30% on all afterburners.
+        * For reference, in 0.9.15, afterburners gained a 50% increase in thrust, 70-80% decrease in fuel usage, and 33% decrease in size.
+	  * Adjusted the Sea Scorpion and Cicada slightly. (@EjoThims)
+	  * Increased ramscoop value of all Remnant ships and the Emergency Ramscoop by 0.75, and adjusted the performance of ramscoops in Ember Waste systems. (@Zitchas)
+    * Graphics:
+      * Added sprites for the Remnant Swan. (@Saugia)
+      * Remodelled Korath Thermal Repeater sprites. (@Daeridanii1)
+      * Remodelled the Android sprite. (@Anarchist2)
+      * Added sprites for a Republic Navy Battleship and a spinal weapon for it. (@GefullteTaubenbrust2)
+      * Reduced the scaling of Hai stations from 0.5 to 0.4. (@Azure3141)
+      * New landscape images for Earth, New China, Chiron, and Solace. (@bene-dictator)
+      * Remodelled the Hai railgun, railgun slug, and railgun slug rack. (@1010todd)
+      * Expanded and improved the Milky Way galaxy sprite. (@Azure3141)
+      * Remodelled the Enforcer Riot Staff and Enforcer Confrontation Gear. (@1010todd)
+      * Added a brown haze. (@RisingLeaf)
+      * Sprites for Hai optical jamming outfits. (@beccabunny)
+      * Added a sprite for a proto-planetary disk. (@RisingLeaf)
+	  * Gave unstable wormholes red link colors. (@Azure3141)
```

Comment:
```suggestion
    * Gave unstable wormholes red link colors. (@Azure3141)
```
this should not be idented, right?

Reply:
It should have 6 spaces, not 4.

Reply:
ok, but why does it display strange?

Reply:
Because it's got a tab in there. ~~I blame VS Code.~~